server/docker-base/Dockerfile

```diff
@@ -581,6 +435,82 @@ COPY template.build.gradle ${EXTENSION_BUILD_GRADLE_TEMPLATE}
 COPY template.gradle.properties ${EXTENSION_GRADLE_PROPERTIES_TEMPLATE}
 COPY template.local.properties ${EXTENSION_LOCAL_PROPERTIES_TEMPLATE}
 
+#
+# iOS + OSX
+#
+
+# For package instructions, see <defold>/scripts/mobile/package_xcode_and_sdks.sh
+# and <defold>/scripts/mobile/package_cctools.sh
+
+ENV LD_LIBRARY_PATH ${LD_LIBRARY_PATH}
+
+# Install flow from  >=1.2.174
+ENV XCODE_12_VERSION      12.5
+ENV MACOS_10_15_VERSION   10.15
+ENV IOS_14_VERSION        14.5
```

Comment:
By updating this version number, we make it so that all our versions that use `IOS_14_VERSION` will now use 14.5. I.e. the ones using 1.2.174 an newer.

I don't think we've discussed this approach before, but we should give it a though, since it might be both good and bad.
Good: That we can update the version for everyone.
Bad: We change things for users that might not suspect it

Reply:
I wonder if there's any radical changes between the 14.x versions?